src/app/satellite.ts

```diff
@@ -15,7 +15,11 @@ export class Satellite {
    }
 	
 	isSpaceDebris(): boolean {
-		return true;
+		if (this.type === "Space Debris") {
```

Comment:
This works great! Just a suggestion: `return this.type === "Space Debris"`

Because the statement already evaluates as a true or false statement, you don't need to add the if/else logic